### DISCUSSION TITLE
Problem at first implementation

### DISCUSSION THREAD

**Discussion by CrisBarrientos** [bootscore/bootscore#594] [Q&A] [answered]
Hello
Can't implement bs-swiper the right way.
I want to put posts that are in the "exposiciones-individuales" category in the swiper.
I put the shortcode in the main page. But all I can see is just one post of that category and apparently a big div empty wrapper.

The installation is in
https://guillermomoscosoartista.cl/v2/wp-admin
user: test
pass: test1test2test3

You think You can help me finding out what is it??
Thanks You!!

**Comment by crftwrk** [comment, marked as answer]
bS Swiper is a plugin for bootScore theme and uses functions from it. You are using Underscores theme, plugin won't work here because there are missing functions.

**Comment by CrisBarrientos** [comment]
Oh, no!! Such a shame! I run into this plugin thru github, I didn't know it uses code from a theme! It seems soo nice, it's a pity that can't work without the theme. Any chance to make it a standalone plugin in the future?? Cause it will be so nice!!!

Well, thank you so much for your answer, sir! You have a good day.

**Reply by crftwrk** [reply]
You can install Bootstrap in your theme and remove following snippets from the files in `template` folder:

```php
<?php if ($categories == 'true') : ?>
  <?php bootscore_category_badge(); ?>
<?php endif; ?>
```

```php
<?php if ('post' === get_post_type()) : ?>
  <p class="meta small mb-2 text-muted">
    <?php
      bootscore_date();
      bootscore_author();
      bootscore_comments();
      bootscore_edit();
    ?>
  </p>
<?php endif; ?>
```

```php
<?php if ($tags == 'true') : ?>
  <?php bootscore_tags(); ?>
<?php endif; ?>
```

Plugin should work now. But you can simply give bootScore theme https://github.com/bootscore/bootscore a try. You will like it.

**Reply by CrisBarrientos** [reply]
That didi it! Thank you so much! Yeah I'm considering this theme in the future, for now and in this job in particular, i just needed a swipe "pagination" of the posts... so, I have to say, you make my day with this tip. Thanks again!!

**Reply by crftwrk** [reply]
You're welcome 👍

You should delete this lines in `main.php` as well to stop the updates if you edited core files:

https://github.com/bootscore/bs-swiper/blob/017957308c7e1ef31544e6417d9797bcff8cf048/main.php#L19-L27

```php
/**
 * Update checker
 */
require 'update/update-checker.php';
$myUpdateChecker = Puc_v4_Factory::buildUpdateChecker(
  'https://bootscore.me/wp-content/plugins/bs-swiper-main/update/plugin.json',
  __FILE__, //Full path to the main plugin file or functions.php.
  'bs-swiper-main'
);
```
